src/routes/withdraw/eth-address-input.tsx

```diff
@@ -45,12 +45,13 @@ export const EthAddressInput = ({
       <button
         type="button"
         onClick={() => setUseConnectedWallet(!useConnectedWallet)}
+        data-testid="enter-wallet-address-link"
         className="button-link fill"
       >
         {useConnectedWallet ? t("enterAddress") : t("useConnectedWallet")}
       </button>
 
-      {isValid ? null : <Callout intent="warn">{t("invalidAddress")}</Callout>}
+      {isValid ? null : <Callout intent="warn" datatest-id="invalid-address-callout" >{t("invalidAddress")}</Callout>}
```

Comment:
chore (blocking): incorrectly specified data test id. As it is incorrect I also assume it is not used. Remove.